### DISCUSSION TITLE
Enpass password manager showing black screes

### DISCUSSION THREAD

**Discussion by abwebster** [rustdesk/rustdesk#5130] [Q&A]
This is a long shot, but perhaps someone knows about this issue. While remotely connected to my cell phone everything works well on rustdesk except for the black screen I get when I start Enpass. Any ideas if this is deliberate  or a bug or something else? If you want to try it install it from play store and remote in and start enpass. Its disappointing.

**Comment by NiceGuyIT** [comment]
Password managers generally don't allow screenshots or screen recording. Here's how you can change that.
<https://support.enpass.io/docs/manual-android/advanced.html#allow-screenshots>

**Reply by abwebster** [reply]
Interesting. It works! Thank you very much. Of note, Enpass linux desktop version has no allow-screenshots box. Its viewable by default from a cell phone rust client.
